src/components/Shared/OTPAuthentication.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState, useEffect, useContext } from 'react'
+import Grid from '@mui/system/Unstable_Grid/Grid'
+import styles from '../../../styles/phoneVerification.module.css'
+import toast from 'react-hot-toast'
+import axios from 'axios'
+import { useAuth } from 'src/hooks/useAuth'
+import { useError } from 'src/error'
+import { ControlContext } from 'src/providers/ControlContext'
+import { AccessControlRepository } from 'src/repositories/AccessControlRepository'
+
+const OTPAuthentication = ({ loggedUser, onClose, window }) => {
+  const [otp, setOtp] = useState(['', '', '', '', '', ''])
+  const [error, setError] = useState('')
```

Comment:
this could be useState([])

Reply:
error is  string it is better if the initial value is ' '